go.sum

```diff
@@ -758,6 +758,8 @@ github.com/hashicorp/golang-lru v0.5.4 h1:YDjusn29QI/Das2iO9M0BHnIbxPeyuCHsjMW+l
 github.com/hashicorp/golang-lru v0.5.4/go.mod h1:iADmTwqILo4mZ8BN3D2Q6+9jd8WM5uGBxy+E8yxSoD4=
 github.com/hashicorp/hcat v0.0.0-20210414201846-1b51a2a2651f h1:L/fJ24aVu/TbYV4AyoMfzir8HCbDDfynXhoZLsLUzrM=
 github.com/hashicorp/hcat v0.0.0-20210414201846-1b51a2a2651f/go.mod h1:AJ2fdGudt/27fXjQBl9srE7mOV9fGZnjVxua/CpjT3I=
```

Comment:
Would running `go mod tidy` remove these and use the added ones below?

Reply:
Good call! It cleaned them up :)